eng/pipelines/templates/Build_and_UnitTest.yml

```diff
@@ -26,6 +26,76 @@ steps:
     inlineScript: |
       Get-ChildItem Env: | Sort-Object Name | Format-Table -Wrap -AutoSize
 
+- task: CredScan@2
```

Comment:
Build_and_UnitTest is on the hot path, therefore making this job slower is going to make the entire pipeline slower. It would be better to create a new job that runs in parallel. Afterwards, we can move the `dotnet-format`, and any other check that can be done independently, to the new job.

This will also avoid CredScan running on both the RTM and non-RTM jobs.

Reply:
I placed on separate pipeline now.